### PR TITLE
created rating for user shows

### PR DESCRIPTION
**Pull request by sunnycake**
@yp0975eu @NateBigStone @ashley7295 @cjdufault Please review and see if it's satisfactory enough. Thank you!

### CODE REVIEW

**Comment by claraj** [comment]
model changes make sense to me although I haven't run the code. Missing unit tests. Modify the ones that test adding notes, and add a new test to ensure note ratings must be one of the 5 valid values. thanks!

**Review by NateBigStone — APPROVED** [review]

File: lmn/models.py [hunk 1]
```diff
@@ -48,10 +48,20 @@ def __str__(self):
...
+        (3, '★★★'),
+        (4, '★★★★'),
+        (5, '★★★★★'),
+    )
+
     show = models.ForeignKey(Show, blank=False, on_delete=models.CASCADE)
     user = models.ForeignKey('auth.User', blank=False, on_delete=models.CASCADE)
     title = models.CharField(max_length=200, blank=False)
     text = models.TextField(max_length=1000, blank=False)
+    rating = models.IntegerField(choices=STAR_RATING, blank=True, default=None)
```
Comment (NateBigStone) [inline lmn/models.py:64]: rating = models.IntegerField(choices=STAR_RATING, blank=False, default=5)

**Comment by yp0975eu** [comment]
This needs tests then it should be good to go.

**Review by yp0975eu — COMMENTED** [review]
Still needs tests

**Review by yp0975eu — CHANGES_REQUESTED** [review]
Still needs tests before merging

**Review by ashley7295 — CHANGES_REQUESTED** [review]
Its looking good, once it contains tests we can get it approved!

**Review by sunnycake — COMMENTED** [review]
@yp0975eu @NateBigStone @ashley7295 I've added unit tests. Let me know if there's anything else I'm missing. I've also updated the time in test_views.py. I was getting assertion error about time 11/27/2020 != 11/28/2020 so I updated that to utc and all tests have passed so far.

**Review by yp0975eu — APPROVED** [review]

**Review by yp0975eu — APPROVED** [review]
:) tests pass. Ratings are working.

**Review by NateBigStone — APPROVED** [review]

**Comment by NateBigStone** [comment]
@sunnycake Sorry, I merged my branch in, and now there's a conflict in lmn/tests/test_forms.py I can push a fix if you can't get to it today

**Comment by sunnycake** [comment]
> @sunnycake Sorry, I merged my branch in, and now there's a conflict in lmn/tests/test_forms.py I can push a fix if you can't get to it today

I'm not sure exactly where the error is. Mind point me to where you're getting the error? Thanks.

File: lmn/tests/test_forms.py [hunk 2]
```diff
@@ -45,10 +45,21 @@ def test_text_too_long_is_invalid(self):
...
+
+    def test_rating_greater_than_five_is_invalid(self):
+        # Rating greater than five is invalid
+        form_data = { "rating" :  6}
+        form = NewNoteForm(form_data)
+        self.assertFalse(form.is_valid())
+
+    def test_ok_title_length_and_rating_is_valid(self):
+        # Title and length is valid and rating is valid with values 1-5
+        form_data = { "title": "blah blah", "text" : "blah, blah, blah.", "rating": 5}
```
Comment (yp0975eu) [inline lmn/tests/test_forms.py:63]: This is missing an assertion

**Review by yp0975eu — CHANGES_REQUESTED** [review]
Just this missing assertion and this should be all set.

**Review by yp0975eu — APPROVED** [review]

**Comment by yp0975eu** [comment]
I think this is good to go :)

File: lmn/tests/test_forms.py (hunk 2 above)
Comment (sunnycake) [reply lmn/tests/test_forms.py:65]: Sorry about that. Thanks for the catch. I've added assertion.

**Pull request merged**